docs/diagrams/PastAppointmentSequenceDiagram.puml

```diff
@@ -36,20 +36,20 @@ create CreatePastAppointmentCommand
 CreatePastAppointmentCommandParser -> CreatePastAppointmentCommand
 activate CreatePastAppointmentCommand
 
-CreatePastAppointmentCommand --> CreatePastAppointmentCommandParser : d
+CreatePastAppointmentCommand --> CreatePastAppointmentCommandParser : appt
 deactivate CreatePastAppointmentCommand
 
 'Hidden arrow to position the destroy marker below the end of the activation bar.
-CreatePastAppointmentCommandParser --> AddressBookParser : d
+CreatePastAppointmentCommandParser --> AddressBookParser : appt
 destroy CreatePastAppointmentCommandParser
 
-AddressBookParser --> LogicManager : d
+AddressBookParser --> LogicManager : appt
 deactivate AddressBookParser
 
 LogicManager -> CreatePastAppointmentCommand : execute()
 activate CreatePastAppointmentCommand
 
-CreatePastAppointmentCommand -> Model : addPastAppointment(d)
+CreatePastAppointmentCommand -> Model : addPastAppointment(appt)
```

Comment:
think that another name besides `appt` could be better since the code adds a `PastAppointment` and not the `CreatePastAppointmentCommand`

Reply:
Fair enough, done!